tests/valid/opticalData/directionWavelengthPolarization.json

```diff
@@ -26,4 +26,4 @@
       }
     }
   ]
```

Comment:
Please create valid and invalid tests to verify with `make shell` and `make test` that your schema works as intended. The tests should cover the entire schema so that unwanted changes can later be detected by failing tests. Please also use `make format`.

Reply:
@c3rromatthias I made the tests valid with two small commits.

Reply:
@c3rromatthias I made hygrothermalData.json valid with a tiny commit and formatted the files. `make test` had no errors.